pkg/k8s/from_yaml.go

```diff
@@ -35,6 +36,7 @@ type DeploymentYml struct {
 	YmlInBytes []byte
 }
 
+// NewDeploymentYml returns a K8s deployment in case there is no error, else returns nil value
```

Comment:
`...returns a K8s deployment yaml....`

Reply:
Same can be done at all other places.
Also let us take view from others.
@princerachit @AmitKumarDas ( Can you please share some insights)

Reply:
Makes sense @sonasingh46